tokio/src/runtime/time/entry.rs

```diff
@@ -43,8 +43,8 @@
 //!
 //! We do, however, also need to track what the expiration time was when we
 //! originally registered the timer; this is used to locate the right linked
-//! list when the timer is being cancelled. This is referred to as the "cached
-//! when" internally.
+//! list when the timer is being cancelled.
+//! This is referred to as the "registered_when" internally.
```

Comment:
If you use backticks, then you do not need to add it to the dictionary.
```suggestion
//! This is referred to as the `registered_when` internally.
```

Reply:
Thanks, I have replaced these quotes with backticks.